test/lib/ufe/ufeTestUtils/ufeUtils.py

```diff
@@ -55,3 +54,18 @@ def selectPath(path, replace=False):
         ufeSelectCmd.replaceWith(selection)
     else:
         ufeSelectCmd.append(sceneItem)
+
+def ufeFeatureSetVersion():
+    '''Return the UFE feature set version taking into account rollback to using
+    0 for unreleased version after we ship a UFE.
+
+    If you need to test for a specific UFE preview release version then check
+    the cmake variable UFE_PREVIEW_VERSION_NUM. This is also set as an env
+    for the python tests.
```

Comment:
Copy-paste damage.

Reply:
Ack, my mistake, got confused with
https://github.com/Autodesk/maya-usd/blob/02637462a15bddd817fc1118c6d10cdfe5db8c61/test/lib/ufe/ufeTestUtils/mayaUtils.py#L173
and clearly they're not the same.